### PR TITLE
Task/add apikey device update

### PR DESCRIPTION
**Pull request by AlvaroVega**
Currently a device could be provisioned with a explicit apikey (i.e. different to group service).
https://github.com/telefonicaid/iotagent-node-lib/blob/master/doc/api.md#device-model

But there is no way to update device apikey using PUT update API
https://github.com/telefonicaid/iotagent-node-lib/blob/master/doc/api.md#put-iotdevicesdeviceid
This PR fixes it

issue https://github.com/telefonicaid/iotagent-node-lib/issues/978

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -1,3 +1,4 @@
+Add apikey to device update
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]: Is there any associated issue to this?

File: lib/services/devices/deviceRegistryMongoDB.js [hunk 2]
```diff
@@ -287,6 +288,7 @@ function update(device, callback) {
             data.endpoint = device.endpoint;
             data.name = device.name;
             data.type = device.type;
+            data.apikey = device.apikey;
```
Comment (fgalan) [inline lib/services/devices/deviceRegistryMongoDB.js:291]: Does documentation any update regarding this? Not sure if we have documented in some place the payload of the update operations, in which case I understand apiKey should be now included.

File: CHANGES_NEXT_RELEASE (hunk 1 above)
Comment (AlvaroVega) [reply CHANGES_NEXT_RELEASE]: No issue reported

File: lib/services/devices/deviceRegistryMongoDB.js (hunk 2 above)
Comment (AlvaroVega) [reply lib/services/devices/deviceRegistryMongoDB.js:291]: apikey currently is in device model: https://github.com/telefonicaid/iotagent-node-lib/blob/master/doc/api.md#device-model
but no full options are describe in update (PUT) request: https://github.com/telefonicaid/iotagent-node-lib/blob/master/doc/api.md#put-iotdevicesdeviceid

Comment (fgalan) [reply lib/services/devices/deviceRegistryMongoDB.js:291]: Good enough. NTC

**Review by fgalan — APPROVED** [review]
LGTM

File: CHANGES_NEXT_RELEASE (hunk 1 above)
Comment (fgalan) [reply CHANGES_NEXT_RELEASE]: I understand at the end we have one, #978, currently added in CNR, so it's fine.

NTC

**Pull request merged**